example/e2e/sanityTest.spec.js

```diff
@@ -16,21 +16,17 @@ describe('NetInfo', () => {
 
   it('should load example app with no errors and show all the examples by default', async () => {
     await expect(element(by.id('examplesTitle'))).toExist();
-    await expect(element(by.id('example-isConnected'))).toExist();
-    await expect(element(by.id('example-currentInfoSingle'))).toExist();
-    await expect(element(by.id('example-currentInfoHistory'))).toExist();
-    await expect(element(by.id('example-isConnectionExpensive'))).toExist();
   });
```

Comment:
Not sure if this is expected or not, but I pulled the repo and checked out this branch, opened it up in VSCode and it blows up seemingly on all the detox e2e tests. If I run `npm test` from the command-line after an `npm i` (I suppose I should use yarn? I dunno) it passes though.

Reply:
How do you mean "blows up"? What's VSCode doing?

Reply:
Bunch of TypeErrors - all identical just different file+line combos, here's text of one: `sanityTest.spec.js example/e2e TypeError: Cannot read property 'reloadReactNative' of null Jest [19,1]`

All the same reloadReactiNative null thing. I'm ignorant of e2e setups in react-native, but given command-line works and VSCode is blowing up trying to run jest (which just worked on the command-line, ugh) it makes me think there is just a tiny mis-config somewhere, such that VSCode is attempting to run e2e things when it should only be attempting unit test things

Reply:
Do you have a Jest plugin for VSCode? It could be trying to run the detox tests there which won't work as it needs the Detox environment.

Reply:
> Do you have a Jest plugin for VSCode

Of course :-), and it's brilliant except this bit of brain-damage - what you suppose is exactly what I'm thinking as well.

Without trying to make "handle random plugins in user editors" a requirement, is there some way to classify Jest tests (I think there is?) and make it so normal jest runs only detect and run non-detox tests? I am keen to learn this entire area of react-native QA but am fully ignorant at the moment, so this suggestion might make no sense or be completely impractical.

Reply:
I've added the `example` folder to the `testPathIgnorePatterns` setting in `jest.config.js`. Hopefully, that should stop VSCode trying to run them.

Reply:
I'll re-test once pushed, thanks!

Reply:
Indeed - that made VSCode+Jest happy, it's 'Jest: :heavy_check_mark:' in my bottom bar now, and no errors in output.